trustfall_core/test_data/tests/parse_errors/transform_first_directive.graphql.ron

```diff
@@ -0,0 +1,12 @@
+TestGraphQLQuery (
+    schema_name: "filesystem",
```

Comment:
There's no way you could have known this (the lack of docs is on me), but the `filesystem` schema is very old and outdated in terms of practices and naming conventions. I've been trying to move all test cases to the newer `numbers` schema since it's newer and better in every way.

You don't have to change this test case if you don't feel like it, we can merge it as-is since I don't want to make you do extra work because of my failure to write docs. Just wanted to mention it so you know for the future.